policy/modules/services/spamassassin.te

```diff
@@ -21,6 +21,15 @@ gen_tunable(spamassassin_can_network, false)
 ## </desc>
 gen_tunable(spamd_enable_home_dirs, false)
 
+## <desc>
+##	<p>
+##	Determine whether spamassassin
+##	can update the rules using th
+##	network.
+##	</p>
+## </desc>
+gen_tunable(spamassassin_can_update, true)
```

Comment:
I think `spamassassin_net_update` or `spamassassin_network_update` would be clearer naming.

Reply:
I can amend that, if you prefer an alternative naming